testng-core/src/test/java/test/reports/ReportTest.java

```diff
@@ -301,8 +301,10 @@ public void reportCreatedWithNullParameter() {
     System.setOut(new PrintStream(systemOutCapture));
     tng.run();
     System.setOut(previousOut);
+    String actual = systemOutCapture.toString();
 
     Assert.assertTrue(
-        systemOutCapture.toString().contains("PASSED: testMethod(null, \"Bazinga!\")"));
+        actual.contains(
+            "PASSED: test.reports.ReportTest$NullParameter.testMethod(null, \"Bazinga!\")"));
```

Comment:
Maybe use another separator symbol between class and method (didn't find if a standard symbol exists)

Reply:
@juherr - Is there any issue with using `.` as the separator here? It seems to be the standard symbol. Even IntelliJ uses that, when you right click a method and say `Copy / Paste Special > Copy reference`. Wouldn't it confuse users if we ended up introducing something else as the separator ?